btr-web/btr-main-app/app/router.options.ts

```diff
@@ -40,7 +40,7 @@ export default <RouterConfig> {
           leftButtons: [],
           rightButtons: []
         },
-        layout: 'business'
+        layout: 'filing'
```

Comment:
we also want this for the route above

Reply:
Added the widget to the business layout.
Also, you can test adding the fee now, as there is inputs. Changing any form input will trigger adding of the fee.

Reply:
can you add this for when the effective date is added as well?